rpm/entry.sh

```diff
@@ -1,11 +1,39 @@
-#!/bin/sh
+#!/bin/bash
 # Entrypoint for the build container to create the rpms and yum repodata:
 
 set -e
-# Download sources if not already available
-cd /root/rpmbuild/SPECS  && spectool -gf kubelet.spec
 
-/usr/bin/rpmbuild --define "_sourcedir /root/rpmbuild/SPECS/" -bb /root/rpmbuild/SPECS/kubelet.spec
+declare -a ARCHS
 
-mkdir -p /root/rpmbuild/RPMS/x86_64/
-createrepo -o /root/rpmbuild/RPMS/x86_64/ /root/rpmbuild/RPMS/x86_64
+if [ $# -gt 0 ]; then
```

Comment:
you should just declare an array or something like this:
```
ARCHS=(
    amd64/x86_64
    arm/armhf
    arm64/aarch64
    ppc64le/?
#  GOARCH/RPMARCH
)
```
Then loop over that array and split the / and go on an build. Only one for loop needed.
This script can be vastly simplified